bin/neat

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env ruby
 
-require "neat"
+require File.dirname(__FILE__) + "/../lib/neat.rb"
```

Comment:
@teoljungberg Just built the gem locally and testing all of this with and without the above change, and it breaks without. This also aligns with the executable in Bourbon and Bitters, so I’m going to run with it.